index.html

```diff
@@ -51,16 +51,19 @@
       <h2>Introduction</h2>
 
 <p>W3C is a <strong>growing and global community</strong> where participants
-choose to work together, and in that process experience differences in
-language, location, nationality, and experience. In such a diverse environment,
+choose to work together, and it is committed to maintaining a positive working environment, where each participant feels appreciated and respected and where everyone adheres to the same high level of standards of personal behavior. In that process we experience differences in language, location, nationality, and experience. In such a diverse environment, 
 misunderstandings and disagreements happen, which in most cases can be resolved
-informally. In rare cases, however, behavior can intimidate, harass, or
-otherwise disrupt one or more people in the community, which W3C will not
-tolerate.</p>
+informally.</p>
+	    <ul>
+<li> W3C community participants should treat each other with respect and professionalism and be mindful of cultural differences.</li>
+<li>W3C community participants should communicate constructively and avoid insulting, unwelcome or demeaning behavior.</li>
+<li>W3C strictly prohibits discrimination, intimidation, harassment, and bullying of any kind and on any basis.</li>
+<li>W3C will not tolerate abusive behavior in any form, whether it is verbal, physical, sexual, or implied.</li>
+		    </ul>
 
 <p>A <strong>Code of Ethics and Professional Conduct</strong> is useful to
 define accepted and acceptable behaviors and to promote high standards of
-professional practice. It also provides a benchmark for self evaluation and
+professional practice. The goal of this code of conduct is to ensure transparency in moderation of the working group and to ensure that the working group is an environment where everyone can participate without fear of harassment.It also provides a benchmark for self evaluation and
```

Comment:
Repeating the abstract word for word seems odd.

Also, I think this paragraph would be clearer if it were directly about the goals of this document:
- define acceptable and expected standards of behaviour
- provide a benchmark
- ensure transparency in managing moderation
- ensure and environment where people can participate without fear of harrassment
- contribute to the identity of the organisation

Reply:
4/4/19 PWECG agreed to incorporate Chaals proposed changes